docs/quickstart.rst

```diff
@@ -2,335 +2,227 @@
+
+Requester
+    The `Requester` entity is composed of a `Publisher` and a `Subscriber` associated with a `RequestTopic` and a `ReplyTopic` respectively.
+    The `Publisher` is in charge of sending the request, while the `Susbscriber` receives the replies.
+
+    To create a `Requester` entity, the ``uxr_buffer_create_requester_xml`` or ``uxr_buffer_create_requester_ref`` shall be used.
+    Once created, requests can be sent through ``uxr_buffer_request``.
+    Replies can be received by sending a data request to the *Agent* with ``uxr_buffer_request_data``,
+    and through the `on_reply` callback which shall be set by the ``uxr_set_reply_callback``.
+    This callback has a parameter :code:`reply_id` which corresponds to the identifier returned by the ``uxr_buffer_request`` call.
+
+Replier
+    The `Reply` entity is a mirror of the `Requester`, that is, it contains a `Publisher` and a `Subscriber` as well,
+    but the topic association is reversed, 
+    as the `Publisher` is associated with the `ReplyTopic` and the `Subscriber` to the `RequestTopic`.
+    In this case, the `Subscriber` is in charge of receiving the request from the `Requester`, while the `Publisher` sends the replies.
+
+    To create a `Replier` entity, the ``uxr_buffer_create_replier_xml`` or ``uxr_buffer_create_replier_ref`` shall be used.
+    Once created, replies can be sent through ``uxr_buffer_reply``.
+    Requests can be received by sending a data request to the *Agent* with ``uxr_buffer_request_data``, 
+    and through the `on_request` callback which shall be set by the ``uxr_set_request_callback``.
+    This callback has a parameter `sample_id` which identifies the request and should be used in the ``uxr_buffer_reply``.
+
+All the files and the code used in this example can be found in the
+`Micro-XRCE-DDS-Client/examples/RequestAdder <https://github.com/eProsima/Micro-XRCE-DDS-Client/tree/master/examples/RequestAdder>`_
+and
+`Micro-XRCE-DDS-Client/examples/ReplyAdder <https://github.com/eProsima/Micro-XRCE-DDS-Client/tree/master/examples/ReplyAdder>`_
+folders.
+
+Requester application
+^^^^^^^^^^^^^^^^^^^^^
+
+Let's now install the *Client* locally, and with the :code:`-DUCLIENT_BUILD_EXAMPLES=ON` flag enabled, so as
+to activate the compilation of the examples. On Linux, this implies running the following: ::
+
+    $ git clone https://github.com/eProsima/Micro-XRCE-DDS-Client.git
+    $ cd Micro-XRCE-DDS-Client
+    $ mkdir build && cd build
+    $ cmake .. -DUCLIENT_BUILD_EXAMPLES=ON
+    $ make
+
+At this point, it's possible to launch the :code:`RequestAdder` executable
+located in the folder :code:`Micro-XRCE-DDS-Client/build/examples/RequestAdder`, which'll make
+the *Client* send two integers as a request, and receive the sum of both integers as a response. ::
+
+    $ Micro-XRCE-DDS-Client/build/examples/RequestAdder/RequestAdder 127.0.0.1 2019
+
+.. TODO: decide which path to give to the user.
+.. TODO: check the command here.
```

Comment:
@pablogs9

Reply:
If we have said `At this point,` I think that we should assume that user is in `build` folder, so the command should be:

```bash
 $ examples/RequestAdder/RequestAdder 127.0.0.1 2019
```

Change if you agree